crossdock/services/query_test.go

```diff
@@ -61,4 +61,13 @@ func TestGetTraces(t *testing.T) {
 
 	_, err = query.GetTraces("bad_svc", "op", map[string]string{"key": "value"})
 	assert.Error(t, err)
+
+	// Test ioutil.ReadAll error
```

Comment:
Same here: perhaps extract this into its own test func.

Reply:
good idea.
And I have changed them following @albertteoh 's example.